tests/test_projects/token_ops/mod.rs

```diff
@@ -0,0 +1,99 @@
+use fuel_tx::Salt;
+use fuels_abigen_macro::abigen;
+use fuels_contract::contract::Contract;
+use rand::rngs::StdRng;
+use rand::{Rng, SeedableRng};
+
+#[tokio::test]
+async fn mint() {
+    abigen!(TestFuelCoinContract, "test_projects/token_ops/src/abi.json",);
+    let salt = new_salt();
+    let compiled = Contract::compile_sway_contract("test_projects/token_ops", salt).unwrap();
+    let (client, fuelcoin_id) = Contract::launch_and_deploy(&compiled).await.unwrap();
+    let fuel_coin_instance = TestFuelCoinContract::new(compiled, client);
+
+    let c = testfuelcoincontract_mod::ContractId {
+        value: fuelcoin_id.into(),
+    };
+
+    let balance_check_1 = ParamsGetBalance {
+        target: fuelcoin_id.into(),
+        asset_id: c.clone(),
+        salt: 1u64, // temp, see: https://github.com/FuelLabs/fuels-rs/issues/89
```

Comment:
Small note about this temp fix, it's been fixed here https://github.com/FuelLabs/fuels-rs/pull/92 but not yet released. I'll make a note to let you know when it's released so we can remove this workaround!

Reply:
sweet, thx!